ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -172,6 +172,11 @@
     reason="Test will not run on LSO deployed cluster"
 )
 
+skipif_no_lso = pytest.mark.skipif(
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L110-L148
Suggestion, can we implement in the similar way we have for other deployments?
Please feel free to ignore if you developed this way in purpose.